core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContext.scala

```diff
@@ -61,6 +62,8 @@ trait AkkaStreamletContext extends StreamletContext {
   private[akkastream] def committableSink[T](outlet: CodecOutlet[T], committerSettings: CommitterSettings): Sink[(T, Committable), NotUsed]
   private[akkastream] def committableSink[T](committerSettings: CommitterSettings): Sink[(T, Committable), NotUsed]
 
+  private[akkastream] def flexiFlow[T](outlet: CodecOutlet[T]): Flow[(immutable.Seq[T], Committable), (Unit, Committable), NotUsed]
```

Comment:
this is the very first API to take a `List`-like element, I wouldn't be shocked to use the `java.util.List` directly, to avoid unnecessary boxing/unboxing when it comes to `javadsl` and, also, for possible performance improvements, does it make sense?